### PR TITLE
Elasticsearch support for change to typeless API

### PR DESCRIPTION
**Pull request by JoelMcKinnon**
To address issues raised in open source community on errors when attempting to install Elasticsearch plugin. 

Author Checklist

- Changes address original issue?	Y
- Unit tests included and/or updated with changes?	Y
- Command line build passes?	Y
- Changes have been smoke-tested?	Y

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -43,9 +43,10 @@
             openmct.legacyRegistry.enable.bind(openmct.legacyRegistry)
         );
 
+        openmct.install(openmct.plugins.Elasticsearch('http://localhost:9200'));
```
Comment (akhenry) [inline index.html:46]: Please revert this change, Open MCT should use LocalStorage by default

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js [hunk 2]
```diff
@@ -103,6 +103,7 @@ define(
 
         // Get a domain object model out of ElasticSearch's response
         ElasticPersistenceProvider.prototype.getModel = function (response) {
+            // console.log('response', response);
```
Comment (akhenry) [inline platform/persistence/elastic/src/ElasticPersistenceProvider.js:106]: Please remove commented code

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js [hunk 3]
```diff
@@ -139,6 +140,7 @@ define(
         };
 
         ElasticPersistenceProvider.prototype.readObject = function (space, key) {
+            // console.log('space, key', space, key)
```
Comment (akhenry) [inline platform/persistence/elastic/src/ElasticPersistenceProvider.js:143]: Please remove commented code

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js [hunk 4]
```diff
@@ -147,7 +149,9 @@ define(
             function checkUpdate(response) {
                 return self.checkResponse(response, key);
             }
-            return this.put(key, value, { version: this.revs[key] })
+            // return this.put(key, value, { version: this.revs[key] })
```
Comment (akhenry) [inline platform/persistence/elastic/src/ElasticPersistenceProvider.js:152]: Please remove commented code.

File: index.html (hunk 1 above)
Comment (JoelMcKinnon) [reply index.html]: done

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js (hunk 2 above)
Comment (JoelMcKinnon) [reply platform/persistence/elastic/src/ElasticPersistenceProvider.js]: done

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js (hunk 3 above)
Comment (JoelMcKinnon) [reply platform/persistence/elastic/src/ElasticPersistenceProvider.js]: done

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js (hunk 4 above)
Comment (JoelMcKinnon) [reply platform/persistence/elastic/src/ElasticPersistenceProvider.js]: done

File: platform/persistence/elastic/src/ElasticPersistenceProvider.js [hunk 5]
```diff
@@ -116,7 +117,9 @@ define(
         // indicate that the request failed.
         ElasticPersistenceProvider.prototype.checkResponse = function (response, key) {
             if (response && !response.error) {
-                this.revs[key] = response[REV];
+                console.log('response, key', response, key)
```
[the remark this answers is not in the document]
Comment (JoelMcKinnon) [reply platform/persistence/elastic/src/ElasticPersistenceProvider.js:120]: done

**Comment by JoelMcKinnon** [comment]
Testing notes: enables installation and usage of ElasticSearch plugin

1. Install and an elasticsearch server locally: https://www.elastic.co/guide/en/elastic-stack-get-started/current/get-started-elastic-stack.html#install-elasticsearch
2. Configure elastisearch.yml for CORS (see https://github.com/nasa/openmct/issues/2886)
3. Comment out localstorage plugin in index.html and install elasticsearch plugin: `openmct.install(openmct.plugins.Elasticsearch('http://localhost:9200'));`
4. Create and remove one or more objects in the tree.

**Comment by akhenry** [comment]
Verified Fixed 05/12/2020
